hermes-management/build.gradle

```diff
@@ -44,21 +44,20 @@ dependencies {
 }
 
 task buildHermesConsole(type: Exec) {
-    workingDir '../hermes-console'
+    def dir = '../hermes-console'
+    workingDir dir
+    outputs.dir(dir + "/dist")
+    inputs.dir(dir + "/static")
     commandLine './package.sh'
 }
 
 task attachHermesConsole(type: Copy, dependsOn: 'buildHermesConsole') {
-    from '../hermes-console/dist/static'
-    into(sourceSets.main.output.resourcesDir.path + '/static')
-}
-
-tasks.register('prepareIndexTemplate') {
+    def srcDir = '../hermes-console/dist/static'
+    def targetDir = sourceSets.main.output.resourcesDir.path + '/static'
```

Comment:
we shouldn't write to the main source directory or copy files in to it.

instead we should output files to a task specific output directory in the `build` folder. then we should add this folder to the sourcesets (similar to line 82 of `hermes-consumers/build.gradle`)